tools/publishIsolate.ts

```diff
@@ -0,0 +1,56 @@
+import fs from 'fs';
+import path from 'path';
+import childProcess from 'child_process';
+import getNpmPackageVersion from 'get-npm-package-version';
+import { compareVersions } from 'compare-versions';
+
+async function main() {
+  const isolatePackageRoot = './dist/isolate';
+
+  const publishPackageList: { packageName: string; dir: string }[] = [];
+
+  // get directory listing
+  const files = fs.readdirSync(isolatePackageRoot);
+  for (const file of files) {
+    const filePath = path.join(isolatePackageRoot, file);
+    const stat = fs.statSync(filePath);
+    if (stat.isDirectory()) {
+      // get package.json
+      const packageJsonPath = path.join(filePath, 'package.json');
+      const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8'));
+      // get version
+      const version = getNpmPackageVersion(packageJson.name);
```

Comment:
`const version = getNpmPackageVersion(packageJson.name) || '1.0.0';`

Js-sdsl will add some new package in the future.